test_list/test.c

```diff
@@ -0,0 +1,157 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "list.h"
+#include <stdbool.h>
+
+
+Node *createNode(int val) {
+    Node *node = (Node *)malloc(sizeof(Node));
+    node -> value = val;
+    node -> next = NULL;
+    return node;
+}
```

Comment:
А почему это здесь, а не в реализации списка???

Плюс со стилем кода что-то вообще не то.
Так не пишут:
```suggestion
Node *createNode(int val) {
    Node *node = (Node *)malloc(sizeof(Node));
    node->value = val;
    node->next = NULL;
    return node;
}
```